Automattic/jetpack-e2e-reports
# Clean up old results

Scope: 
* Clean up older results

Testing instructions:
* Locally, run script `DAYS=n ./bin/cleanup.sh`. It should remove all folders/files inside `docs` that are older than `n` days and regenerate the reports for remaining folders that had files removed.
* Workflow should run every day at 10 UTC with DAYS=30
